pkg/kubectl/cmd/BUILD

```diff
@@ -65,9 +160,6 @@ go_library(
         "version.go",
     ],
     tags = ["automanaged"],
-    visibility = [
-        "//build/visible_to:pkg_kubectl_cmd_CONSUMERS",
-    ],
```

Comment:
Please leave this.  

See https://github.com/kubernetes/kubernetes/blob/master/build/visible_to/README.md